pages/events/index.vue

```diff
@@ -157,10 +181,33 @@ export default class extends QiskitPage {
     img: '/images/events/no-events.svg'
   }
 
+  // multiselect
+  regionOptions = this.getOptions(this.regions)
+  typeOptions = this.getOptions(this.types)
+  theme: string = 'light'
+  labelRegions: string = 'All locations'
+  labelTypes: string = 'All types'
+  feedback: string = 'fixed'
+
   get noEvents (): boolean {
     return (this as any).filteredEvents.length === 0
   }
 
+  getOptions (optionsList): Array<object> {
```

Comment:
`object` is very generic, create a type for this `object`. See an example [here](https://github.com/Qiskit/qiskit.org/blob/master/components/landing/TheQuickStart/StartLocally.vue#L78) and [here](https://github.com/Qiskit/qiskit.org/blob/master/components/landing/TheQuickStart/StartLocally.vue#L84)

Reply:
Updated - any feedback would be appreciated :pray: